README.md

```diff
@@ -0,0 +1,118 @@
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
+# Enurable
+
+A project to create the Ruby enumerable module.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+## Clone this repository
+
+  $ git@github.com:NduatiKagiri/enumerable.git
+  $ cd enumerable
+
+## Run Project
+  $ Kali Rolling WSL
+
+### Install
+  $ rbenv install 3.0.1
+  $ rbenv global 3.0.1
+  $ ruby -v
+
+### Deployment
+
+[Front-end Netlify Deployment](https://airbnb.nduatikagiri.co.ke//)
+
+[Back-end Heroku Deployment](https://fast-wildwood-74017.herokuapp.com/)
+
+
+## Kanban Board
+
+### Link
+
+[GitHub Project](https://github.com/NduatiKagiri/airbnb-backend/issues) 
+
+<!-- ### Initial State Project Link -->
+[GitHub First Issue](https://github.com/NduatiKagiri/airbnb-backend/issues)
+
+### Final Team members
+
+- John Nduati Kagiri
+- Brenda Wihogora
+- Jesse Uzoma
+
+
+## Authors
+
+
+
+👤 **Uzoma Jesse**
+
+- GitHub: [@jesse](https://github.com/Brenda309)
+- Twitter: [@jesse](https://twitter.com/BrendaWihogora)
+- LinkedIn: [jesse uzoma](https://linkedin.com/in/BrendaWihogora/)
+
+
+👤 **John Nduati Kagiri**
+
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-john-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/NduatiKagiri/)
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-@ba104781-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/)
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-john-white?logo=LinkedIn&logoColor=1DA1F2&style=plastic)]()
+
+👤 **Brenda Wihogora**
+
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-brenda-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/Brenda309)
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-brenda-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/)
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-brenda-white?logo=LinkedIn&logoColor=1DA1F2&style=plastic)](https://www.linkedin.com/in//)
+
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Splash Background Image by [T. Kaiser](https://unsplash.com/@tkaiser)
+- Add Car Background Image by [Jorgen Hendriksen](https://unsplash.com/@jor9en)
+- Design inspired by [Murat Korkmaz](https://www.behance.net/muratk)
+- Hat tip to anyone whose code was used.
+- Inspiration.
+- Our supportive families and Microverse colleagues.
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪